src/main/twirl/gitbucket/core/repo/files.scala.html

```diff
@@ -33,6 +33,7 @@
     <div class="head" style="height: 24px;">
       <div class="pull-right">
         <div class="btn-group">
+          <a href="@{helpers.url(repository)}/archive/@{helpers.encodeRefName(branch)}@if(pathList.length > 0){/@pathList.mkString("/")}/@{helpers.encodeRefName(branch)}.zip" class="btn btn-sm btn-default"><i class="octicon octicon-cloud-download"></i> Download ZIP</a>
```

Comment:
Moving "Download" button to this button group is understandable because it now supports downloading sub tree. It's not displayed for mobile originally.

Reply:
@takezoe Do mobile users need downloads?

Reply:
I don't think so. So I suggest add "pc" style class to this button.

Reply:
Ok, I will hide it for mobile.